controllers/replace_misconfigured_pods_test.go

```diff
@@ -387,6 +387,63 @@ var _ = Describe("replace_misconfigured_pods", func() {
 		})
 	})
 
+	When("PVC name doesn't match", func() {
+		It("should need a removal", func() {
+			pvc := corev1.PersistentVolumeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "Test-storage",
+					Labels: map[string]string{
+						fdbtypes.FDBInstanceIDLabel:   instanceName,
+						fdbtypes.FDBProcessClassLabel: string(fdbtypes.ProcessClassStorage)},
+					Annotations: map[string]string{
+						fdbtypes.LastSpecKey: "f0c8a45ea6c3dd26c2dc2b5f3c699f38d613dab273d0f8a6eae6abd9a9569063",
+					},
+				},
+			}
```

Comment:
We could also use `internal.GetPvc(cluster, processClass, idNum)` here instead of creating it by ourself, the same for the other test cases.

Reply:
👍